app/views/pageflow/admin/users/_may_delete.html.erb

```diff
@@ -0,0 +1,6 @@
+<p class="may_delete">
+  <%= t('pageflow.admin.users.me.delete_account_hint') %>
+    <%= link_to(t('pageflow.admin.users.me.delete_account'),
+                delete_me_admin_users_path,
+                :class => 'delete_account') %>
```

Comment:
In all fairness, they do not explicitly advertise checking embedded Ruby on their home page https://houndci.com/